MidtransKit/MidtransKit/VTClassHelper.h

```diff
@@ -12,6 +12,9 @@
 
 #import <MidtransCoreKit/VTHelper.h>
 
+extern NSString *const _TRANSACTION_SUCCESS;
+extern NSString *const _TRANSACTION_FAILED;
```

Comment:
There is no technical reason for this, it is only my preferred way. If you have better idea, please tell me 🙏